src/openforms/prefill/utils.py

```diff
@@ -0,0 +1,18 @@
+from typing import Any
+
+from glom import Path, PathAccessError, glom
+
+
+def find_in_dicts(*dicts: dict[str, Any], path: Path) -> str | None:
```

Comment:
this return type is also wrong - the JSON can contain `null` or `number`, and that is what would be returned. So I think you can only say `JSONValue` as type annotation

Furthermore, using `None` to signify a missing value is ambiguous, since `null` in JSON can be defined but that will be returned as `None` and then ignored. It's better to rely on the glom exception here or use a special sentinel value to signify the key is missing.

Reply:
This was removed as we decided to use `glom` for it.